Makefile

```diff
@@ -1,7 +1,8 @@
 CLEANUP_FILES  = tarantool.log
-CLEANUP_FILES += *.xlog*
+CLEANUP_FILES += *.xlog
+CLEANUP_FILES += *.vylog
```

Comment:
To be honest I'm not sure that it's really needed. AFAIU all vylog files are placed in 51* folders. Also vinyl produces run and index files but they are not mentioned here.